nntrainer/graph/network_graph.cpp

```diff
@@ -552,23 +562,23 @@ NetworkGraph::getGraph(const std::string &input_layer,
   /** count layers after output layer */
   unsigned int num_layers_remove_end = 0;
   if (!output_layer.empty()) {
-    for (auto iter = layers.rbegin(); iter != layers.rend(); iter++) {
-      if ((*iter)->getName() != output_layer)
+    for (auto iter = adj.rbegin(); iter != adj.rend(); iter++) {
+      if ((*iter).front().layer->getName() != output_layer)
         num_layers_remove_end++;
       else
         break;
     }
   }
 
-  if (num_layers_remove_end == layers.size())
+  if (num_layers_remove_end == adj.size())
     return {};
 
   /** count layers before input layer */
   unsigned int num_layers_remove_start = 0;
   if (!input_layer.empty()) {
-    for (auto iter = layers.begin();
-         iter != layers.end() - num_layers_remove_end; iter++) {
-      if ((*iter)->getName() != input_layer)
+    for (auto iter = adj.begin(); iter != adj.end() - num_layers_remove_end;
+         iter++) {
+      if ((*iter).front().layer->getName() != input_layer)
```

Comment:
istrequal here?

Reply:
Followup on this.

Reply:
https://github.com/nnstreamer/nntrainer/pull/1016/commits/55dab436b8e7be5c59b7a68a8cf3601e4406fd25#r595745132